components/divider/stories/divider.stories.js

```diff
@@ -1,3 +1,4 @@
+import { html } from "lit";
 import { Template } from "./template";
 
 /**
```

Comment:
I didn't [migrate anything about the icon sizes](https://opensource.adobe.com/spectrum-css/divider.html#migrationguide). Let me know if I should add that.